newsroom/news_api/app/__init__.py

```diff
@@ -43,7 +43,8 @@ def get_app(config=None):
               data=SuperdeskDataLayer,
               settings=app_config)
 
-    app._general_settings = {}
+    app._general_settings = {'news_api_time_limit_days': {'type': 'number',
```

Comment:
`news_api_time_limit_days` is defined in `news_api/__init__.py`. If you  add `newsroom.news_api` to the `CORE_APPS` in `settings.py` this setting will be registered in one place instead of two.

Reply:
It's only defined in the parameters passed to a Newsroom app, I'm not sure how I can get to that!